<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1" colspan="4">
       <widget class="QLabel" name="labelDirectory">
        <property name="text">
         <string>Directory: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Axis On</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Point Size: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonLoad">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSpinBox" name="spinBoxCloudNo"/>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Cloud No. </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="MyGlWidget" name="widgetRender" native="true"/>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxPointSize">
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGlWidget</class>
   <extends>QWidget</extends>
   <header location="global">myglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
